Rachellombardi website redesign (#688)
CHTC/chtc-website-source: _uw-research-computing/htc-overview.md

---
highlighter: none
layout: guide
title: HTC System Overview
guide:
    order: 0

>     category: Job Submission

Reviewer: ```suggestion
    category: Basics and Policies
```

1. [High-Throughput Computing at CHTC](#high-throughput-computing-at-chtc)  
2. [HTC User Policies](#htc-system-user-policies)
    - [HTC System Specific Policies](#htc-system-specific-policies)    
4. [HTC Hardware and Configuration](#htc-hardware-and-configuration)
    - [HTC Operating System and Hardware](#htc-operating-system-and-software)   
    - [HTC Sumbmit Servers](#submit-servers)     
    - [HTC Execute Nodes](#execute-nodes)   
    - [HTC Operating System and Software](#operating-system-and-software)
6. [Data Storage and Management](#data-storage-and-management)   
    - [Tools for managing /home and /staging space](#tools-for-managing-home-and-staging-space)

{% endcapture %}
{% include /components/directory.html title="Table of Contents" %}

<span name="high-throughput-computing-at-chtc"></span>
# High-Throughput Computing at CHTC

The CHTC high-throughput computing (HTC) cluster provides support a variety of computational research tasks. The HTC system offers CPUs/GPUs, high-memory nodes, and other specialized hardware. Workflows that run well on this system include RNA/DNA sequencing, machine learning workflows, weather modeling, monte carlo simulations, etc. 

> To get access to the HTC Cluster, please complete our

Reviewer: ```suggestion
To get access to the HTC System, please complete our
```

    longer to start than usual. <u> If you plan on submitting over
    1000 jobs per submit file, we ask that you ensure each job has a
    minimum run time of 5 minutes (on average). <u>
-   **The default disk quota is 20 GB** in your `/home` directory, as a 
    starting point. You can track your use of disk space and your quota value,
    using our [Quota Guide](check-quota.html). If you need more space
    for concurrent work, please send an email to chtc@cs.wisc.edu.
-   **Submitting jobs with \"large\" files:** HTCondor\'s
    normal file transfer mechanism (\"transfer\_input\_files\") is good for 
    files up to 100MB in size (or 500MB total, per job). For jobs with larger
    files, please see our guide on [File Availability](file-availability.html) 
    Options, and contact us to make arrangements.

<span name="htc-hardware-and-configuration"></span>

# HTC Hardware and Configuration

The HTC System consists of several submit servers and many compute (aka execute) 
nodes. All users log in at a login node, and submit their workflow as HTCondor jobs that run on execute points. 

<span name="htc-operating-system-and-software"></span>

## HTC Operating System and Software

> Submit servers in the HTC System are running CentOS 8 Stream Linux. 

Reviewer: ```suggestion
Submit servers in the HTC System are running CentOS 7 Linux. 
```

Due to the distributed and independent nature of the HTC system's execute points, there can be a variety of operating systems on the pool of execution point resources (especially for users that opt into running jobs on the globally available OSPool operated by the OSG). However, the default operating system is CentOS 8 Stream Linux unless users request to run on a different operating system using their HTCondor submit file. 

The HTC system is a test bed for the [HTCondor Software Suite](https://htcondor.org/), and thus is typically running the latest or soon-to-be-released versions of HTCondor. 

> To see more details of other software on the cluster, see the [HTC Software page](htc-software.html). 

Reviewer: ```suggestion
To see more details of other software on the cluster, see our [HTC Guides page](guides#htc-documentation). 
```

<span name="submit-servers"></span>

## HTC Submit Servers

There are multiple submit servers for the HTC system. The two most common submit servers are `submit1.chtc.wisc.edu` and `submit2.chtc.wisc.edu`. All users will be notified what submit server they should log into when their account is created.

> For more details on logging in, see the "Connecting to CHTC" guide linked above. 

Reviewer: Is this linked above?

Author: no. deleting the sentence for now.

> For more details on logging in, see the "Connecting to CHTC" guide linked above. 

Author: ```suggestion
```
